src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -32,6 +32,10 @@
   public static final int QUESTION_FETCH_DATETIME = 5;
   public static final int QUESTION_FETCH_NUMBEROFFOLLOWERS = 7;
   public static final int QUESTION_FETCH_NUMBEROFANSWERS = 11;
+  public static final int QUESTION_FETCH_USERFOLLOWSQUESTION = 12;
+
+  // Constant used to check if the user follows a question.
+  public static final int QUERY_SET_USERID = 1;
```

Comment:
I think the variable name doesn't fit the comment description and is too general. You could change it to `QUESTION_QUERY_SET_USERID` or something similar.

Reply:
you're right, done